examples/cloud_run_cloudevents/Dockerfile

```
@@ -0,0 +1,17 @@
+# Use the official Python image.
+# https://hub.docker.com/_/python
+FROM python:3.7-slim
+
+# Copy local code to the container image.
+ENV APP_HOME /app
+ENV PYTHONUNBUFFERED TRUE
+
+WORKDIR $APP_HOME
+COPY . .
+
+# Install production dependencies.
+RUN pip install gunicorn cloudevents functions-framework
+RUN pip install -r requirements.txt
+
+# Run the web service on container startup.
+CMD exec functions-framework --target=hello --signature-type=cloudevent
```

We should use the exec syntax, this will also mean the functions-framework is pid 1 and receiving process signals like SIGTERM.

```suggestion
CMD ["functions-framework", "--target=hello", "--signature-type=cloudevent"]
```